server/assets.go

```diff
@@ -53,6 +53,16 @@ func Assets(opts AssetsOpts) http.Handler {
 				WithField("url", r.URL).
 				Info("Serving assets")
 		}
-		assets.Handler().ServeHTTP(w, r)
+		up := URLPrefixer{
+			Prefix: basepath,
+			Next:   assets.Handler(),
+			Attrs: [][]byte{
+				[]byte(`src="`),
+				[]byte(`href="`),
+				[]byte(`url(`),
+				[]byte(`data-basepath="`), // for forwarding basepath to frontend
```

Comment:
Nope, this is for fixing up the font URLs in the CSS that comes back like:

```css
@font-face {
  font-family: "A font";
  src: url('/path/to/font');
}
```

This probably should be smarter about reading the Mime type of the data being written and reacting accordingly, but since there's no overlap in the attributes right now, I think it's okay for the time being.